remotespark/RemoteSparkMagics.py

```diff
@@ -49,11 +46,6 @@ def __init__(self, shell, data=None, test=False):
         except KeyError:
             self.logger.error("Could not read env vars for serialization.")
 
-        if use_auto_viz:
```

Comment:
This is a setting in the default_config.json file as well as in Constants. Can you please remove it from there too?